_includes/tutorials/transforming/confluent/markup/dev/run-producer.adoc

```diff
@@ -0,0 +1,13 @@
+In a new terminal, run:
+
+```
+ccloud kafka topic produce raw-movies \
```

Comment:
In the staging site, the following lines are coming out formatted strangely.  Maybe convert this to a `.sh` file and include that file...like shown above (that would also follow other adoc patterns)

Reply:
I've broken this out into a separate `.sh` file in commit https://github.com/confluentinc/kafka-tutorials/pull/848/commits/34c69403818b635bab2b219cda65a66043aef8b3 and that has resolved the strange rendering (verified locally) :)